tests/unity_catalog/test_delta_sharing_cli.py

```diff
@@ -527,6 +550,12 @@ def test_update_recipient_cli(api_mock, echo_mock):
                 '8.8.8.8',
                 '8.8.4.4'
             )
+        },
+        'properties_kvpairs': {
+            'properties': [
+                {"key": "k1", "value": "v1"}, 
+                {"key": "k2", "value": "v2"}
+            ]
```

Comment:
For my info, do you know why this is double nested?

Reply:
It is because we reuse a common proto message definition on unity catalog service but we need to un-apply `(json_inline) = true` setting in order to allow the client to delete all existing properties. See more in the REST API PR: https://github.com/databricks/universe/pull/227235/files#diff-9d05ccad757e3c0f1a3d894d62996a64b3ab4df3704c0f55290f21f7e0fb2549L85